src/base_classes/active_space_solver.cc

```diff
@@ -591,6 +602,26 @@ RDMs ActiveSpaceSolver::compute_avg_rdms_ms_avg(
     return RDMs(true, g1a, g2ab, g3aab);
 }
 
+std::map<StateInfo, std::string> ActiveSpaceSolver::dump_wave_function() {
+    std::map<StateInfo, std::string> out;
+
+    std::string method_lowercase(method_);
+    std::transform(method_lowercase.begin(), method_lowercase.end(), method_lowercase.begin(),
```

Comment:
There is a function to do this in `helpers/string_algorithms.h`

Reply:
Thanks, changes made.